app/assets/stylesheets/components/_notice.scss

```diff
@@ -0,0 +1,28 @@
+.app-c-notice {
+  clear: both;
+  @include responsive-bottom-margin;
+  padding: $gutter-two-thirds;
+  border: 5px solid $govuk-blue;
+
+  @include media(mobile) {
+    padding: $gutter-half;
+  }
+
+  p:last-child {
```

Comment:
As @nickcolley pointed out, this should be explicit that it's styling something within the govspeak component.